dev-cpp/ms-gsl/ms-gsl-2.0.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
```

Comment:
vcs-snapshot seems unnecessary, this is no snapshot and has a fairly predictable `S` dirname.

Reply:
Leftover from before upstream tagged any releases. Now removed.